### PR TITLE
Add VirtualMedia Information in the API Readme

### PR DESCRIPTION
**Pull request by sunithasomanna**
Add detailed information on the following VirtualMedia APIs in the API readme:

/redfish/v1/Managers/{ManagerId}/VirtualMedia
/redfish/v1/Managers/{ManagerId}/VirtualMedia/{VirtualMediaID}
/redfish/v1/Managers/{ManagerId}/VirtualMedia/{VirtualMediaID}/Actions/VirtualMedia.InsertMedia
/redfish/v1/Managers/{ManagerId}/VirtualMedia/{VirtualMediaID}/Actions/VirtualMedia.EjectMedia

### CODE REVIEW

**Review by bharath-b-hpe — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -6289,16 +6285,188 @@ curl -i GET \
...
+curl -i GET \
+   -H 'Authorization:Basic {base64_encoded_string_of_[username:password]}' \
+ 'https://{odim_host}:{port}/redfish/v1/Managers/{ManagerId}/VirtualMedia/{VirtualMediaID}'
+```
+
+
+>**Sample response body**
+
+```
+{
```
Comment (amar-shalgar) [inline docs/README.md:6361]: Response body is incorrect. Please correct it

File: docs/README.md [hunk 2]
```diff
@@ -6289,16 +6285,188 @@ curl -i GET \
...
+  "WriteProtected":true
+}' \
+ 'https://{odimra_host}:{port}/redfish/v1/Managers/{ManagerId}/VirtualMedia/{VirtualMediaID}/Actions/VirtualMedia.InsertMedia'
+
+```
+
+>**Sample response body**
+
+```
+{
```
Comment (amar-shalgar) [inline docs/README.md:6432]: This response got changed, Please update

File: docs/README.md [hunk 3]
```diff
@@ -6289,16 +6285,188 @@ curl -i GET \
...
+ 'https://{odimra_host}:{port}/redfish/v1/Managers/{ManagerId}/VirtualMedia/{VirtualMediaID}/Actions/VirtualMedia.EjectMedia'
+```
+
+<blockquote>NOTE: No payload is required for this operation. </blockquote>
+
+>**Sample response body**
+
+
+```
+{
```
Comment (amar-shalgar) [inline docs/README.md:6476]: This response got changed, pls update

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address comments

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md:6361]: addressed

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md:6432]: addressed

File: docs/README.md (hunk 3 above)
Comment (sunithasomanna) [reply docs/README.md:6476]: addressed

**Review by amar-shalgar — APPROVED** [review]
Virtual media details looks OK

**Pull request merged**